src/main/webapp/video.js

```diff
@@ -45,3 +43,31 @@ function onYouTubeIframeAPIReady() {
 function onPlayerReady(event) {
   event.target.playVideo();
 }
+
+/** Starts timer if {@code currState} is playing, stops otherwise. */
+function startStopTimerOnStateChange(currState) {
+  const state = currState.data;
+  if (state == window.YT.PlayerState.PLAYING) {
+    startTimer();
+    return;
+  }
+  stopTimer();
+}
+
+/** Starts timer which calls {@code sync} with the current video time.  */
```

Comment:
This comment describes the code, not the functionality. For me to understand what this actually does I would need to look up the JSDoc for sync.

Reply:
fixed